.travis.yml

```diff
@@ -55,3 +59,5 @@ jobs:
       python: 3.6
       env: PURPOSE='Automation Docker'
       if: repo = Azure/azure-cli and type = push
+  allow_failures:
+    - env: PURPOSE='Lint Command Table and Help'
```

Comment:
It's questionable why we whould really want a build stage that allows failures. I think historically, these get totally ignored.

Reply:
So can this go away now?